### PR TITLE
Added check of params for template's methods events and helpers

### PR DESCRIPTION
**Pull request by dolgarev**
Hello guys!

I use Сoffeescript and faced unpleasant trouble recently, when a small bug made me feel nerveous since I couldn't figure out its reason and Meteor didn't suggest anything. 

I wrote a code, where passed function instead of object  Event Map:

```js
Template.myJobsInfo.events ->
'click [data-action="check_status"]': -> console.log 'OK'
 ```

I did not receive mssage about error, but my code still did not work. As appeared Meteor does not check parameters, which are passed to methods events and helpers.

My minipatch allows to discard this undefined behavior.

### CODE REVIEW

**Comment by dolgarev** [comment]
Hello again!
I have joined to MCA. What should I do next?

**Comment by mitar** [comment]
+1

**Comment by stubailo** [comment]
...thanks!

File: packages/blaze/template.js [hunk 1]
```diff
@@ -437,6 +437,10 @@ Blaze.TemplateInstance.prototype.subscriptionsReady = function () {
  * @param {Object} helpers Dictionary of helper functions by name.
  */
 Template.prototype.helpers = function (dict) {
+  if (Object.prototype.toString.call(dict) !== "[object Object]") {
```
Comment (mitar) [inline packages/blaze/template.js]: Not sure why not using `_.isObject` here though.

Comment (stubailo) [inline packages/blaze/template.js]: Yeah that would be better.

...submit a PR?

Comment (stubailo) [inline packages/blaze/template.js]: :stuck_out_tongue:

Comment (mitar) [inline packages/blaze/template.js]: Give me commit bit. :-P

Comment (stubailo) [inline packages/blaze/template.js]: Hmmm, thinking about it.

**Pull request merged**